applications/HDF5Application/custom_io/hdf5_model_part_io_base.h

```diff
@@ -72,16 +72,12 @@ class ModelPartIOBase : public IO
 
```

Comment:
I am a bit lost on why you define a HDF5 namespace with the modelpart_io base class in it. It will hide the kratos one no?

Reply:
i guess it does. It is just that i am not used to this way of doing (not a problem though)